src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/RemoveAnnotationConflictQuickFix.java

```diff
@@ -41,11 +50,14 @@
  * @author Angelo ZERR
  *
  */
-public class RemoveAnnotationConflictQuickFix {
-    private final String[] annotations;
+public abstract class RemoveAnnotationConflictQuickFix implements IJavaCodeActionParticipant {
 
+    private static final String ANNOTATION_TO_REMOVE = "annotationsToRemove";
```

Comment:
Should probably name this constant `ANNOTATIONS_TO_REMOVE` since the value is a list and there can be more than one.

Reply:
Yes, I changed to **ANNOTATIONS_TO_REMOVE**